src/Discord.Net.Commands/Attributes/Preconditions/RequireBotPermissionAttribute.cs

```diff
@@ -52,7 +53,7 @@ public override async Task<PreconditionResult> CheckPermissionsAsync(ICommandCon
                 if (guildUser == null)
                     return PreconditionResult.FromError("Command must be used in a guild channel");
```

Comment:
What about a second custom message for this one?

Reply:
Can add more properties to set, but I wasn't sure about doing that.